scanners/zap-advanced/.helm-docs.gotmpl

```diff
@@ -26,6 +26,9 @@ usecase: "WebApp & OpenAPI Vulnerability Scanner extend with authentication feat
 
 {{- define "extra.chartAboutSection" -}}
 ## What is OWASP ZAP?
+:::caution Deprecation Notice
+The `zap-advanced` and `zap` ScanType are being deprecated in favor of the `zap-automation-framework`, which encompasses all functionalities of the previous ScanTypes. We recommend transitioning to "zap-automation-framework". This change will take effect in the upcoming release cycle. For guidance on migrating to "zap-automation-framework," please refer to [here](/docs/scanners/zap-automation-framework#migration-to-zap-automation-framework).
```

Comment:
I think this would be a bit clearer :)

```suggestion
The `zap-advanced` and `zap` ScanType are being deprecated in favor of the `zap-automation-framework`, which encompasses all functionalities of the previous ScanTypes. We recommend transitioning to the "zap-automation-framework" as soon as possible. `zap-advanced` and `zap` ScanTypes will be removed take in the upcoming v5 release. For guidance on migrating to "zap-automation-framework," please refer to [migration to zap-automation framework](/docs/scanners/zap-automation-framework#migration-to-zap-automation-framework).
```